src/services/publish.service.ts

```diff
@@ -38,6 +39,100 @@ export class PublishService extends Repository {
       configureOptions.posting_latitude = lat.toString();
       configureOptions.posting_longitude = lng.toString();
     }
-    return await this.client.media.configure(configureOptions);
+    return await this.client.media.configureTimeline(configureOptions);
```

Comment:
This method doesn't exists.
Please, `npm run typescheck` before commiting or pushing :)

Reply:
BTW WebStorm have very convenient tool for npm scripts
https://www.jetbrains.com/help/webstorm/npm-tool-window.html